### DISCUSSION TITLE
account link functionality in the account table view

### DISCUSSION THREAD

**Discussion by ChristianKirchnerIlmenau** [nuxsmin/sysPass#1800] [Ideas]
Hi guys,

I use that beautiful tool every day very often because I'm responsible to create new systems and store some credentials and document/link this in the project documentations.

So it was beautiful for me if I could use a Direct-Link Button inside of the acount table views to share that account in the project documentation.

The current solution for me is to click to an account to pick up the direct link at the bottom of the site.

It is possible to implement this?

**Comment by nuxsmin** [comment]
Hello, I'm glad that you found sysPass useful :)

Regarding the direct link on the search page, yes it could be achieved but I need to check the performance impact, because every link is signed using HMAC in order to check the integrity.

Regards.

**Comment by ChristianKirchnerIlmenau** [comment]
Thanks for taking care and work on my request.
Will hope that the performance aspects will not behind this.
Let me know if you have a new status about it.
